plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -134,6 +136,18 @@ export function AboutCard(props: AboutCardProps) {
     }
   }
 
+  // todo: should this be a constant in a scaffolder package?
+  if (entity.metadata.annotations?.['backstage.io/source-template']) {
+    try {
+      sourceTemplateRef = parseEntityRef(
+        entity.metadata.annotations?.['backstage.io/source-template'],
+      );
```

Comment:
Should this also do a check to see if the template still exists and can be created from it? To handle cases where templates are removed.

Reply:
It's a good thought, but it would require an extra lookup when the card gets rendered.

Reply:
I suppose it might also be good to do this anyway, from an authz perspective.